multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py

```diff
@@ -313,7 +322,8 @@ def create_transaction_for_setting_special_role_on_semi_fungible_token(
         add_role_nft_create: bool,
         add_role_nft_burn: bool,
         add_role_nft_add_quantity: bool,
-        add_role_esdt_transfer_role: bool
+        add_role_esdt_transfer_role: bool,
+        add_role_esdt_modify_creator: bool = False
```

Comment:
we need more roles also for semi fungible token

Reply:
added the missing roles